docker/misk-web/Dockerfile

```diff
@@ -1,20 +1,21 @@
 FROM node:alpine
 
-RUN apk add --no-cache tini python python2 alpine-sdk
+RUN apk add --no-cache tini
 
 WORKDIR /tmp/web/
 
 # Populate yarn cache by installing all @misk/ packages
 RUN yarn add \
     --pure-lockfile \
     --non-interactive \
+    --modules-folder /node_modules \
```

Comment:
What's populating this folder? `alpine`?

Reply:
`yarn add` is installing the core `@misk/` packages and all their sub-dependencies into `/node_modules` directory.